ros_ws/src/autonomous/wallfollowing2/script/wallfollowing.py

```diff
@@ -90,7 +100,9 @@ def find_left_right_border(points, margin_relative=0.1):
 
 
 def follow_walls(left_circle, right_circle):
-    predicted_car_position = Point(0, 1.5)
+    global last_speed
+
+    predicted_car_position = Point(0, 1.4 + last_speed)
```

Comment:
May give the magic number `0.14` a name or add a comment describing what the magic number does.